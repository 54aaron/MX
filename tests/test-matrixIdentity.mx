int main(){
    Matrix m;
    Matrix n;
    Matrix f;
    m = identity(2);
    n = identity(3);
    f = identity(6);
    prints("==============================================");
    prints(" 2x2 Identity Matrix ");
    print_matrix(m);
    prints("==============================================");
    prints(" 3x3 Identity Matrix ");
    print_matrix(n);
    prints("==============================================");
    prints(" 6x6 Identity Matrix ");
    print_matrix(f);
    prints("==============================================");
}
