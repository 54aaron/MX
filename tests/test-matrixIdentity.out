==============================================
 2x2 Identity Matrix 

[ 1, 0 ]
[ 0, 1 ]
==============================================
 3x3 Identity Matrix 

[ 1, 0, 0 ]
[ 0, 1, 0 ]
[ 0, 0, 1 ]
==============================================
 6x6 Identity Matrix 

[ 1, 0, 0, 0, 0, 0 ]
[ 0, 1, 0, 0, 0, 0 ]
[ 0, 0, 1, 0, 0, 0 ]
[ 0, 0, 0, 1, 0, 0 ]
[ 0, 0, 0, 0, 1, 0 ]
[ 0, 0, 0, 0, 0, 1 ]
==============================================
